### ISSUE TITLE
Your repository is abusive

### ISSUE THREAD

**Issue by againstRacism** [FiloSottile/age#129]
1. Rename disgusting 'master' branch to some other name (e.g. 'main')
2. Change 'identity' to 'equality'. Identity means that you are better than me, it's abusive
3. Rename your language. I'm a disabled person, I can't walk. 'Go' is abusive to me.
4. Remove link to abusive repository: `github.com/str4d/rage`. Rage is abusive.
5. Remove abusive words from your wordlist: `https://github.com/FiloSottile/age/blob/80c6b4edd9c5cf5e0e98d8c3687076c842efca6a/cmd/age/wordlist.go`
    * above
    * abuse
    * master
    * female, woman (there are more than 2 genders, why you not included them?!)
    * male, man
    * hurt
    * hunt
    * upset
    * rubber
    * garbage
    * fat
    * damage
    * cruel
    * control
    * awful 
    * awkward 
    * blame

Until you make this changes - your repository is abusive to me and I can't use your project!

**Comment by alerque** [comment]
No. Concepts like 'abuse' and 'racism' are context dependent, raw usage of words in a different context does not imply those things. Calling somebody 'fat' can be an insult, calling a file system 'FAT' means it is a specific family of file system with a File Allocation Table. You cannot simply search for the presence of words and call a project abusive based on the presence of English words with no context.

**Comment by Tharre** [comment]
Don't feed the troll.
